virtcontainers/device/drivers/vfio.go

```diff
@@ -128,6 +141,15 @@ func (device *VFIODevice) Detach(devReceiver api.DeviceReceiver) (retErr error)
 		}
 	}()
 
+	if device.GenericDevice.DeviceInfo.ColdPlug {
+		// nothing to detach, device was cold plugged
+		deviceLogger().WithFields(logrus.Fields{
+			"device-group": device.DeviceInfo.HostPath,
+			"device-type":  "vfio-passthrough",
+		}).Info("Nothing to detach. VFIO device was cold plugged")
```

Comment:
Question for my own clarity:

Theoretically a container/pod could be created with a device attached, and then the device could be removed as part of an update later on, right?

Is this limitation on our ability to hot-remove a cold-plugged device?